h/views/client.py

```diff
@@ -50,23 +116,27 @@ def annotator_token(request):
     return generate_jwt(request, 3600)
 
 
-@view_config(route_name='embed')
-def embed(context, request):
-    request.response.content_type = b'text/javascript'
-    request.response.text = client.render_embed_js(
-        assets_env=request.registry['assets_client_env'],
-        app_html_url=request.route_url('widget'),
-        base_url=request.route_url('index'))
-    return request.response
+@view_config(route_name='embed',
+             renderer='h:templates/embed.js.jinja2')
+def embed(request):
+    request.response.content_type = 'text/javascript'
+
+    assets_env = request.registry['assets_client_env']
+    base_url = request.route_url('index')
```

Comment:
`base_url` and `service_url` happen to have the same URL but they have different meanings. The first is the _root URL to which asset URLs are relative_, the second is the URL which the client should use to construct URLs that link into pages in the service.